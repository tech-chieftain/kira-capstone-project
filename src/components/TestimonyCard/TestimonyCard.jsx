import PropTypes from "prop-types";

import Card from "react-bootstrap/Card";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

import Rating from "react-rating";

import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faStar as fasStar, faQuoteLeft, faQuoteRight } from "@fortawesome/free-solid-svg-icons";
import { faStar as farStar } from "@fortawesome/free-regular-svg-icons";

import SmallProfileCard from "../SmallProfileCard/SmallProfileCard";

const TestimonyCard = ({ imgSrc, quote, rating, color, name, job, profilePicture }) => (
  <Card>
    <Row>
      <Col>
        <Card.Img variant="top" src={imgSrc} />
      </Col>
      <Col className="align-self-center">
        <Card.Body>
          <Card.Text className="text-center">
            <div className="text-start">
              <FontAwesomeIcon icon={faQuoteLeft} color={color} size="2x" />
            </div>
            {quote}
            <div className="text-end">
              <FontAwesomeIcon icon={faQuoteRight} color={color} size="2x" />
            </div>
            <Rating
              readonly
              initialRating={rating}
              emptySymbol={<FontAwesomeIcon icon={farStar} />}
              fullSymbol={<FontAwesomeIcon icon={fasStar} />}
              fractions={2}
            />
          </Card.Text>
        </Card.Body>
        <Card.Body>
          <SmallProfileCard name={name} job={job} ProfilePicture={profilePicture} />
        </Card.Body>
      </Col>
    </Row>
  </Card>
);

TestimonyCard.propTypes = {
  imgSrc: PropTypes.string,
  quote: PropTypes.string,
  rating: PropTypes.number,
  color: PropTypes.string,
  name: PropTypes.string,
  job: PropTypes.string,
  profilePicture: PropTypes.string,
};

TestimonyCard.defaultProps = {
  imgSrc: "https://picsum.photos/300/200",
  quote: "",
  rating: 4.5,
  color: "",
  name: "",
  job: "",
  profilePicture: "",
};

export default TestimonyCard;
