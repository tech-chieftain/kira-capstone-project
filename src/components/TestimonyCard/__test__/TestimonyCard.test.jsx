import * as React from "react";
import * as renderer from "react-test-renderer";
import TestimonyCard from "../TestimonyCard";

it("renders TestimonyCard without props correctly", () => {
  const tree = renderer.create(<TestimonyCard />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders TestimonyCard with props correctly", () => {
  const tree = renderer
    .create(
      <TestimonyCard
        imgSrc="https://picsum.photos/300/200"
        quote="To be or not to be?"
        rating={3.5}
        color="#008080"
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
