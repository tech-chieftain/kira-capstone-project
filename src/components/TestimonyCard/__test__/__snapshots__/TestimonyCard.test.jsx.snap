// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TestimonyCard with props correctly 1`] = `
<div
  className="shadow w-100 d-flex justify-content-start card"
>
  <div
    className="p-0 m-0 row"
  >
    <div
      className="p-0 m-0 col"
    >
      <img
        className="card-img-top"
        src="https://picsum.photos/300/200"
      />
    </div>
    <div
      className="align-self-center col"
    >
      <div
        className="card-body"
      >
        <p
          className="text-center card-text"
        >
          <blockquote
            className="blockquote"
          >
            <div
              className="text-start"
            >
              <svg
                color="#008080"
                fill="currentColor"
                height="2rem"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": "#008080",
                  }
                }
                viewBox="0 0 512 512"
                width="2rem"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M464 256h-80v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8c-88.4 0-160 71.6-160 160v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48zm-288 0H96v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8C71.6 32 0 103.6 0 192v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48z"
                />
              </svg>
            </div>
            <p>
              To be or not to be?
            </p>
            <div
              className="text-end"
            >
              <svg
                color="#008080"
                fill="currentColor"
                height="2rem"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": "#008080",
                  }
                }
                viewBox="0 0 512 512"
                width="2rem"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
                />
              </svg>
            </div>
          </blockquote>
          <footer
            className="blockquote-footer"
          >
            <small
              className="text-muted"
            >
              
               
              <span
                className="fst-italic fw-light"
              >
                
              </span>
            </small>
            <div>
              <span
                style={
                  Object {
                    "direction": "ltr",
                    "display": "inline-block",
                  }
                }
              >
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={Object {}}
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "50%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={Object {}}
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "0%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
              </span>
            </div>
          </footer>
        </p>
      </div>
      <div
        className="text-center card-body"
      />
    </div>
  </div>
</div>
`;

exports[`renders TestimonyCard without props correctly 1`] = `
<div
  className="shadow w-100 d-flex justify-content-start card"
>
  <div
    className="p-0 m-0 row"
  >
    <div
      className="p-0 m-0 col"
    >
      <img
        className="card-img-top"
        src="https://picsum.photos/700/300"
      />
    </div>
    <div
      className="align-self-center col"
    >
      <div
        className="card-body"
      >
        <p
          className="text-center card-text"
        >
          <blockquote
            className="blockquote"
          >
            <div
              className="text-start"
            >
              <svg
                color="#008080"
                fill="currentColor"
                height="2rem"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": "#008080",
                  }
                }
                viewBox="0 0 512 512"
                width="2rem"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M464 256h-80v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8c-88.4 0-160 71.6-160 160v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48zm-288 0H96v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8C71.6 32 0 103.6 0 192v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48z"
                />
              </svg>
            </div>
            <p>
              
            </p>
            <div
              className="text-end"
            >
              <svg
                color="#008080"
                fill="currentColor"
                height="2rem"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": "#008080",
                  }
                }
                viewBox="0 0 512 512"
                width="2rem"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
                />
              </svg>
            </div>
          </blockquote>
          <footer
            className="blockquote-footer"
          >
            <small
              className="text-muted"
            >
              
               
              <span
                className="fst-italic fw-light"
              >
                
              </span>
            </small>
            <div>
              <span
                style={
                  Object {
                    "direction": "ltr",
                    "display": "inline-block",
                  }
                }
              >
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "visibility": "hidden",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  onClick={[Function]}
                  onMouseMove={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  style={
                    Object {
                      "cursor": "inherit",
                      "display": "inline-block",
                      "position": "relative",
                    }
                  }
                >
                  <span
                    style={Object {}}
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595zm-7.247.47H6.72L8 2.507 6.72 6.722H2.348zm3.537 2.75l-1.307 4.305 1.307-4.305zm7.767-2.75H9.28h4.372zm-8.75.9h2.366L8 5.214l.732 2.41h2.367l-1.915 1.49.731 2.409L8 10.032l-1.915 1.49.731-2.41-1.915-1.49z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    style={
                      Object {
                        "display": "inline-block",
                        "left": 0,
                        "overflow": "hidden",
                        "position": "absolute",
                        "top": 0,
                        "width": "0%",
                      }
                    }
                  >
                    <svg
                      fill="currentColor"
                      height="28px"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="28px"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M9.595 6.252L8 1 6.405 6.252H1l4.373 3.4L3.75 15 8 11.695 12.25 15l-1.623-5.348L15 6.252H9.595z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </span>
              </span>
            </div>
          </footer>
        </p>
      </div>
      <div
        className="text-center card-body"
      />
    </div>
  </div>
</div>
`;
